kitchen.dokken.yml

```diff
@@ -32,11 +27,6 @@ platforms:
       image: dokken/centos-stream-8
       pid_one_command: /usr/lib/systemd/systemd
 
-  - name: centos-stream-9
```

Comment:
There's no point removing these files here, they're templated in. 

We should try and support stream-9 over centos-7 in the future too.

Reply:
i create a new issue for this, i'll do this in a separate PR
https://github.com/sous-chefs/ntp/issues/244